src/resources/i18n/translations/en.json

```
@@ -186,7 +186,7 @@
   "ARIA LABEL - Queue Next": "Enqueue as next",
   "Audio livestreams": "Audio livestreams",
   "Auto": "Auto",
-  "Auto download by default on subscribe": "Turn on auto-downloading for a podcast when you subscribe to it.",
+  "Auto download by default on subscribe": "When subscribing.",
   "Auto Downloading Off": "Auto-downloading off",
   "Auto Downloading On": "Auto-downloading on",
   "Auto-enqueue downloaded episodes": "Auto-enqueue downloaded episodes",
```

```suggestion
  "Auto-enqueue downloaded episodes": "Auto-add downloaded",
```
This is the one in Settings → Queue. (Assuming it is also for clips).
Is it auto-downloaded too? If not, "Download and play" as an extra button would be better?